frontend/styles/components/calendar/WeekStrip.module.scss

```
@@ -0,0 +1,65 @@
+.weekday {
+    font-size: 11px;
+    font-weight: 600;
+    color: $medium-grey-color;
+    text-transform: uppercase;
+}
+
+.circle {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    // Matches MiniCalendar's fixed 32px day circles minus the 2px ring width below, so a
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)